src/Components/NavBar/NavBar.tsx

```diff
@@ -116,6 +116,7 @@ export const NavBar: React.FC = track(
             {isLoggedIn && (
               <>
                 <NavItem
+                  onHover={false}
```

Comment:
One should be able to exclude this prop since `false` and `undefined` (missing the prop) both evaluate to false in the end.